profiles/aws/env/charts/cert-manager.yaml

```diff
@@ -0,0 +1,8 @@
+charts:
```

Comment:
I see that these redundant file creates a lot of overhead. Can you create one override file per provider so that we can use one set of demo files? I would hate to maintain 3 identical copies, but most of all this refactoring gives us the opportunity to clearly show in these override files what is different per provider. Much easier to inspect.

Reply:
makes sense, will do so

Reply:
I managed to extract a common part for all cloud vendors!
Only settings.*.yaml and cluster.yaml are cloud specific.
I was able to do after refactoring DNS settings.